src/styles/index.scss

```
@@ -1,3 +1,117 @@
 body {
   margin: 0;
+  font-family: "Roboto, sans-serif";
+}
+
+$color-blue-accent: #00acdc;
+$color-main-accent: #060b35;
+$color-gray: #616070;
+$card-width: 200px;
```

It is better to use more universal naming for variables.
Also you can export the variables to a separate variable file.
Example:

```suggestion
$accent-color: #00acdc;
$main-color: #060b35;
$second-color: #616070;
$card-width: 200px;
```